### PR TITLE
Up2053463

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: code/smash mouth.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
+Print("Somebody once told me the world is gonna roll me
```
Comment (Ryan-Ursell) [inline code/smash mouth.py:1]: Shouldn't print be lowercase?

**Comment by SamuelHelps** [comment]
+2

File: code/smash mouth.py (hunk 1 above)
Comment (SamuelHelps) [reply code/smash mouth.py:1]: no

Comment (ghost) [reply code/smash mouth.py:1]: 1. no
2. anti yes
3. nah

Comment (cbrittonjsy) [reply code/smash mouth.py:1]: How do you print?

Comment (Ryan-Ursell) [reply code/smash mouth.py:1]: You don't.

**Pull request merged**